lib/order.rb

```diff
@@ -1,7 +1,28 @@
-require './lib/menu.rb'
+
 
 class Order
+  attr_reader :ordered, :menus, :order_total
+
+  def initialize
+    @menus = {"Curry" => 12, "Sushi" => 4, "Edemame" => 2, "Burrata" => 6, "Salad" => 10}
+    @ordered = []
+    @order_total = 0
+end
+
+def display_menu
+  @menus.each {|key, value|
+  puts "#{key} : £#{value}"}
+end
```

Comment:
Lines 6-14 - I really appreciate this code as it is easy to read and I wish I used it especially for display-menu